Ginger/GingerCoreCommon/Actions/ActInputValue.cs

```diff
@@ -146,13 +146,13 @@ public ObservableList<dynamic> ListDynamicValue
 
         private string mValueForDriver;
 
-        public string ValueForDriver { get { return mValueForDriver; } set { mValueForDriver = value; OnPropertyChanged(Fields.ValueForDriver); } }
+        public string ValueForDriver { get { return mValueForDriver; } set { mValueForDriver = value; OnPropertyChanged(ValueForDriver); } }
 
         //TODO: fix me soemthing wrong here !!!!!!!!!!!!!!!!!!!!!!!!!!!!!! - not needed - commented out - delete me later
         // when delete - reading exisitng flow will not load the Value!!!!!!!!!!!!!!!! be careful
         //private string mStoreToVariable;
-        [IsSerializedForLocalRepository]
-        public virtual string StoreToVariable { get { return mValue; } set { mValue = value; OnPropertyChanged(Fields.ValueForDriver); } }
+        //[IsSerializedForLocalRepository]
```

Comment:
Removing serialize is fine but removing the field itself will break when loading existing xml no ?

Reply:
its not breaking. i have tested it.

Reply:
if it was tested and working - delete all commented lines

Reply:
done